ocdskingfisherprocess/database.py

```diff
@@ -442,6 +442,53 @@ def store_collection_file_errors(self, collection_id, file_name, url, errors):
                 'errors': errors,
             })
 
+    def get_releases_to_check(self, collection_id, override_schema_version=None):
+        data = {'collection_id': collection_id}
+        sql = """ SELECT
+                           release_with_collection.id,
+                           release_with_collection.data_id,
+                           release_with_collection.package_data_id
+                           FROM release_with_collection"""
+        if override_schema_version:
+            sql += """ LEFT JOIN release_check ON release_check.release_id = release_with_collection.id
+                          AND release_check.override_schema_version = :override_schema_version
+                       LEFT JOIN release_check_error ON release_check_error.release_id = release_check_error.id
+                          AND release_check_error.override_schema_version = :override_schema_version """
+            data['override_schema_version'] = override_schema_version
+        else:
+            sql += """ LEFT JOIN release_check ON release_check.release_id = release_with_collection.id
+                       LEFT JOIN release_check_error ON release_check_error.release_id = release_check_error.id """
```

Comment:
I think these should have 
```
AND release_check.override_schema_version is null
```

As there is a risk that the script will pick up the overridden_schema_versions releases/records.

Reply:
Oh yes - good point!

I think this bug didn't cause any issues before because we do the checks without a schema version before the checks with one - but there is nothing to guarantee that will be true in the future, so we should fix this. Thanks.